### ISSUE TITLE
Quick pick items with icons issue - Theia 1.47

### ISSUE THREAD

**Issue by safisa** [eclipse-theia/theia#13492]
### Bug Description:
Hi,
I have upgraded my Theia extension to the latest 1.47.1 version from 1.45.1, and I got this regression issue when running this command: Terminal --> Run Task...
I think it is related to the quick-pick component.

Here is the issue in screenshots:

Theia (latest 1.47.1):
![image](https://github.com/eclipse-theia/theia/assets/10339204/28cab8bc-bc1f-4f09-9fe5-7f87c7d254ba)

My custom Theia (1.47.1):
![image](https://github.com/eclipse-theia/theia/assets/10339204/f2db0ca3-f957-4f61-866c-4fef8402dad9)

My custom Theia (1.45.1):
![image](https://github.com/eclipse-theia/theia/assets/10339204/6d0eab7a-f13f-4e6c-a6fb-44fd2c37e3e4)

The issue does not happen in Theia browser example, only on my Theia, I don't know what is causing it.
Can anyone help and guide me, where and what can I check to fix it?

Thanks in advance

### Additional Information

- Operating System: Mac
- Theia Version: Theia 1.47.1

**Comment by JonasHelming** [comment]
It very likely has something to do with the recent Monaco upgrade. https://github.com/eclipse-theia/theia/issues/13449 looks related, maybe this helps?

**Comment by safisa** [comment]
@JonasHelming I found that the issue is related to codicon 0.0.27. I saw that the latest Theia is using version 0.0.35, and mine was using 0.0.27, and there was a breaking change in 0.0.28 (Update display property to use inline-block) that causes this issue.
I am not sure why my build did not bring the same codicon version, but once I deleted the yarn.lock file, it brings the latest one. Maybe it is because the Theia core package uses @vscode/codicons: "*" ...
